app/src/app.js

```diff
@@ -97,27 +97,23 @@ app.use(koaLogger());
 
 koaValidate(app);
 
+RWAPIMicroservice.register({
```

Comment:
I might be missing something, but I see you removed the Promise rejection handling - does a failed registration get correctly logged in case something's wrong (i.e. missing env var :shrug:)?

Reply:
This actually lead me down another rabbit hole, and I did a refactor of the lib.